src/pykx/pandas_api/pandas_meta.py

```diff
@@ -153,6 +153,37 @@ def mean(self, axis: int = 0, numeric_only: bool = False):
             tab
         )
 
+    @api_return
+    def kurt(self, axis: int = 0, numeric_only: bool = False):
+        tab = self
+        if 'Keyed' in str(type(tab)):
+            tab = q('{(keys x) _ 0!x}', tab)
```

Comment:
In this [pr](https://github.com/KxSystems/pykx/pull/16), it was said that it was better to use `q.value(tab)`

Reply:
Done